registration-processor/core-processor/registration-processor-uin-generator-stage/src/main/java/io/mosip/registration/processor/stages/uingenerator/stage/UinGeneratorStage.java

```diff
@@ -159,6 +161,9 @@ public class UinGeneratorStage extends MosipVerticleAPIManager {
 	@Value("${mosip.regproc.uin.generator.dob.log.enable:false}")
 	private boolean dobLogEnable;
 
+	@Value("${mosip.regproc.max.retrycount}")

```

Comment:
Add uin.generator also after mosip.regproc to follow standard naming convention for regproc